templates/openstack/install/index.html

```diff
@@ -0,0 +1,69 @@
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <div>
+        <h1>Install OpenStack</h1>
+        <p>Try out OpenStack on a single machine or start building a production cloud on a cluster: you choose.</p>
+      </div>
+    </div>
+    <div class="col-6">
+      <img src="{{ ASSET_SERVER_URL }}95212c08-OpenStack_Logo_2016.svg" alt="" class="u-align--center u-vertically-center u-hide--small" width="275" />
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep is-bordered">
+  <div class="row">
+    <div class="col-12 p-card--highlighted p-divider u-equal-height">
+      <div class="col-6 p-divider__block">
+        <p class="p-heading--five">Single machine</p>
+        <h2>Try OpenStack on a single&nbsp;machine</h2>
+        <div class="p-card__content">
+          <p>Conjure-up is an ideal tool for people who want to build an OpenStack cloud on a single machine in minutes. Using LXD containers, this OpenStack deployment perfectly models a production cluster on real machines.</p>
+          <p><a href="/openstack/install/try-openstack">Install OpenStack on a localhost&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <p class="p-heading--five">Cluster</p>
+        <h2>Build your own production&nbsp;cloud</h2>
+        <div class="p-card__content">
+          <p>The same conjure-up utility also deploys your production OpenStack cloud across a rack of physical servers using <a href="/server/maas">MAAS</a> and the underlying model enables you to operate OpenStack efficiently at scale.</p>
+          <p><a href="/openstack/install/build-openstack">Deploy OpenStack on a cluster of physical machines&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<div class="p-strip is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}1f1d581a-picto-quote-orange.svg" width="200" alt="" />
+    </div>
+    <div class="col-7 prefix-1 u-vertically-center">
+      <div class="row">
+        <div class="col-12">
+          <h2>Need more help?</h2>
+          <p>Let our cloud experts help you take the next step.</p>
+          <p>
+            <a class="p-button--positive" href="https://www.ubuntu.com/openstack/contact-us?product=cloud-openstack">Contact us</a>
+          </p>
+          <p>Already a Landscape Dedicated Server customer? Upgrading is simple, see the instructions in the <a class="p-link--external" href="https://help.landscape.canonical.com/LDS/ReleaseNotes">release notes</a>.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{% include "download/server/_other-ways.html" %}
+
+{% include "shared/contextual_footers/_contextual_footer.html" with first_item="_cloud_bootstack" second_item="_download_cloud_bootstack" third_item="_download_documentation" %}
```

Comment:
wrong middle contextual_footer

should be 

**Growing your cloud?**
Ubuntu Advantage enterprise support agreements cover OpenStack and include the Landscape licenses needed for the Autopilot in larger cloud deployments.
[ Buy now ]
Or contact our sales team ›